Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -82,17 +82,24 @@ def tests
       opts = []
 
       if ENV["CI"]
-        opts << "--combine-stderr"
-        opts << "--serialize-stdout"
+        opts += %w[
+          --combine-stderr
+          --serialize-stdout
+          --multiply-processes 2
+        ]
+      else
+        opts += %w[
+          --nice
+        ]
       end
 
-      args = [
-        "--color",
-        "-I", HOMEBREW_LIBRARY_PATH/"test",
-        "--require", "spec_helper",
-        "--format", "progress",
-        "--format", "ParallelTests::RSpec::RuntimeLogger",
-        "--out", "tmp/parallel_runtime_rspec.log"
+      args = %W[
+        --color
+        -I #{HOMEBREW_LIBRARY_PATH}/test
```

Comment:
This will fail if there are spaces in `HOMEBREW_LIBRARY_PATH`. Unlikely, but just sayin'.